pkg/testcase/daemonset.go

```diff
@@ -31,7 +32,7 @@ func TestDaemonset(deployWorkload bool) {
 	)
 	nodeNames, err := shared.RunCommandHost(cmd)
 	if err != nil {
-		return
+		GinkgoT().Errorf(err.Error())
```

Comment:
Use `shared.ReturnLogError` here ?

Reply:
i am only adding that on the low level functions for now.